docs/source/logos.rst

```diff
@@ -3,29 +3,35 @@
 Images and Logos
 ================
 
+Here are some commonly used Dask icons and logos
+(see the `Dask style guide <dask.org/style-guide>`_ for more details).
```

Comment:
Sorry, one last nit -- without `https://www.` sphinx attempts to make this a relative url (which doesn't exist)

```suggestion
(see the `Dask style guide <https://www.dask.org/style-guide>`_ for more details).
```